packages/app/src/vis-packs/core/complex/MappedComplexVis.tsx

```diff
@@ -95,6 +87,15 @@ function MappedComplexVis(props: Props) {
         invertColorMap={invertColorMap}
         abscissaParams={axisMapping[dimMapping.indexOf('x')]}
         ordinateParams={axisMapping[dimMapping.indexOf('y')]}
+        alpha={
+          visType === ComplexVisType.PhaseAmplitude
+            ? {
+                array: amplitudeArray,
+                domain: amplitudeDomain,
+                label: 'Amp.',
+              }
+            : undefined
+        }
```

Comment:
I changed the signature of `Heatmap` to constrain that `alphaArray` and `alphaDomain` must be either both supplied or both `undefined`.

With the previous signature, we could have `alphaArray` defined and `alphaDomain` `undefined` leading to a silent failure: https://h5web-docs.panosc.eu/?path=/story/visualizations-heatmapvis--alpha&args=alphaDomain:!null

Reply:
Great idea 💯